xgi/classes/hypergraph.py

```diff
@@ -1154,20 +1154,6 @@ def max_edge_order(self):
             d_max = 0 if self._node else None
         return d_max
 
-    def edges_of_order(self, d):
```

Comment:
Nice but should replace the instances where it is used in the the two .py files in`drawing`.

Reply:
Good catch, done!